src/pudl/metadata/templates/metadata.yml.jinja

```diff
@@ -0,0 +1,108 @@
+title: Catalyst Cooperative Data Portal (EXPERIMENTAL)
+description_html: |
+  <p>This <a href="https://datasette.io">Datasette</a> instance publishes
+  several databases produced by
+  <a href="https://catalyst.coop">Catalyst Cooperative</a> as part of the
+  <a href="https://github.com/catalyst-cooperative/pudl">Public Utility Data
+  Liberation (PUDL) Project</a>.
+  <h2>Caution:</h2>
+  <ul>
+  <li>The structure of the data and the API are not necessarily stable, so don't
+  build any critical infrastructure on top of this just yet.</li>
+  <li>If you find something wrong, please
+  <a href="https://github.com/catalyst-cooperative/pudl/issues">make an issue
+  on GitHub</a> to let us know.</li>
+  <li>You can reach the PUDL team at
+  <a href="mailto:pudl@catalyst.coop">pudl@catalyst.coop</a>.</li>
+  <li>If you find our work useful, please consider
+  <a href="https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=PZBZDFNKBJW5E&source=url">making
+  a recurring contribution</a> to support it.</li>
+  </ul></p>
+license: {{ license.name }}
+license_url: {{ license['path'] }}
+databases:
+  pudl:
+    source: PUDL GitHub Repository
+    source_url: https://github.com/catalyst-cooperative/pudl
+    about: {{ datasources['pudl'].title }}
+    about_url: {{ datasources['pudl'].path }}
+    license: {{ datasources['pudl'].license_pudl.name }}
+    license_url: {{ datasources['pudl'].license_pudl.path }}
+    description_html: |
+      <p>{{ datasources['pudl'].description  }}</p>
+      <p>This database contains cleaned and normalized data from...
+      <ul>
+      <li><a href="https://www.eia.gov/electricity/data/eia860/">EIA Form 860</a> (2001-2020)</li>
+      <li><a href="https://www.eia.gov/electricity/data/eia860m/">EIA Form 860m</a> (through 2021-08)</li>
+      <li><a href="https://www.eia.gov/electricity/data/eia923/">EIA Form 923</a> (2001-2020)</li>
+      <li><a href="https://www.ferc.gov/industries-data/electric/general-information/electric-industry-forms/form-1-electric-utility-annual">FERC Form 1</a> (1994-2020)</li>
```

Comment:
I think in theory we should be able to generate this list of links with titles and available years from the `DataSource` metadata. We'll want to do something similar in the data source documentation exports too.